packages/client/components/TeamPrompt/TeamPromptTopBar.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import useRouter from '~/hooks/useRouter'
+import {TeamPromptTopBar_meeting$key} from '~/__generated__/TeamPromptTopBar_meeting.graphql'
+import {meetingAvatarMediaQueries} from '../../styles/meeting'
+import BackButton from '../BackButton'
+import {AvatarGroupBlock, HeadingBlock, MeetingTopBarStyles} from '../MeetingTopBar'
+import TeamPromptOptionsMenu from './TeamPromptOptionsMenu'
+
+const TeamPromptHeaderTitle = styled('h1')({
+  fontSize: 16,
+  lineHeight: '24px',
+  margin: 0,
+  padding: 0
+})
+
+const TeamPromptHeader = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'flex-start'
+})
+
+const ButtonContainer = styled('div')({
+  alignItems: 'center',
+  alignContent: 'center',
+  display: 'flex',
+  height: 32,
+  marginLeft: 11,
+  position: 'relative',
+  [meetingAvatarMediaQueries[0]]: {
+    height: 48,
+    marginLeft: 10
+  },
+  [meetingAvatarMediaQueries[1]]: {
+    height: 56
+  }
+})
+
+interface Props {
+  meeting: TeamPromptTopBar_meeting$key
+}
+
+const TeamPromptTopBar = (props: Props) => {
+  const {history} = useRouter()
+  const {meeting: meetingRef} = props
+
+  const meeting = useFragment(
+    graphql`
+      fragment TeamPromptTopBar_meeting on TeamPromptMeeting {
+        name
+        ...TeamPromptOptionsMenu_meeting
+      }
+    `,
+    meetingRef
+  )
+
+  const {name: meetingName} = meeting
+
+  return (
+    <MeetingTopBarStyles>
+      <HeadingBlock>
+        <TeamPromptHeader>
+          <BackButton ariaLabel='Back to Meetings' onClick={() => history.push('/meetings')} />
+          <TeamPromptHeaderTitle>{meetingName}</TeamPromptHeaderTitle>
+        </TeamPromptHeader>
+      </HeadingBlock>
+      <AvatarGroupBlock>
+        <ButtonContainer>
+          <TeamPromptOptionsMenu meetingRef={meeting} />
+        </ButtonContainer>
+      </AvatarGroupBlock>
+      {/* :TODO: (jmtaber129): Add avatars, overflow menu, etc. */}
```

Comment:
+1 I guess that's need a little update, we already have a menu :)

Reply:
Updated + moved into the avatar block